generic/htx_test.py

```diff
@@ -151,19 +151,12 @@ def stop_htx(self):
         """
         self.log.info("shutting down the %s ", self.mdt_file)
         cmd = 'htxcmdline -shutdown -mdt %s' % self.mdt_file
-        process.system(cmd, ignore_status=True)
+        process.system(cmd, timeout=120, ignore_status=True)
 
         daemon_state = process.system_output('/etc/init.d/htx.d status')
         if daemon_state.decode().split(" ")[-1] == 'running':
             process.system('/usr/lpp/htx/etc/scripts/htxd_shutdown')
 
-    def tearDown(self):
-        """
-        tearDown
-        """
-        if self.failed:
```

Comment:
@naveen538 You can remove the flag `self.failed` also from the test, since that is not needed anymore.

Reply:
@narasimhan-v changes are done as requested.